services/121-service/src/utils/phone-number.helpers.ts

```diff
@@ -5,18 +5,12 @@
  * @returns The formatted phone number, including "+"-prefix
  */
 export function formatPhoneNumber(phoneNumber: string): string {
-  if (!phoneNumber) {
-    throw new Error('No phone number provided');
-  }
-
   // Strip all (possibly existing) prefixes
   const onlyNumbers = phoneNumber.replace(/\D/g, '');
-
-  // What is left, could be invalid already
+  // If result is empty, then do not add a "+"-prefix
   if (!onlyNumbers) {
-    throw new Error('No valid phone number provided');
+    return onlyNumbers;
```

Comment:
This is not the promised outcome of this helper-method.

"only the digits" is not "a formatted phone-number in the shape of: `+0000000000`".

If you need the phone-number as as formatted string, you should use this helper. If you don't want that, you should _not_ use this helper.

Reply:
You're right that it doesn't align with what's promised. So I changed the return description now.

I do stand by the dropping of the error throwing given that we need to allow empty phoneNumber sometimes now given this new feature.